test/integration/multinode_test.go

```diff
@@ -65,6 +66,26 @@ func TestMultiNode(t *testing.T) {
 			})
 		}
 	})
+
+	// sub-test for deploying app.
+	t.Run("serial", func(t *testing.T) {
```

Comment:
it is not needed to start another cluster here, DeployApp2Nodes already runs after FreshStart2Nodes and will deplouy on that .

so we u can delete this one